EpamWebSiteTestingFramework/Epam.TestAutomation.Pages/PageObjects/Pages/JobListingsPage.cs

```diff
@@ -0,0 +1,111 @@
+    public TextInput KeywordInputField => new(By.XPath("//*[@id='new_form_job_search_1445745853_copy-keyword']"));
+
+    public Dropdown LocationsDropdown =>
+        new Dropdown(By.XPath("//*[@class='select2-selection select2-selection--single']"));
+
+    public ElementsList LocationsDropdownList => new(By.XPath("//*[@class='select2-results__option']"));
+    public Panel DropdownLocationsPanel => new Panel(By.XPath("//*[@class='select2-results__options open']"));
+
+    public Button FindButton => new(By.XPath("//*[@type='submit']"));
+
+    public Button SelectedLocationChosen(string cityName) =>
+        new Button(By.XPath($"//*[@class='select2-selection__rendered'][contains(text(), '{cityName}')]"));
+
+    public Button ChosenCityLine =>
+        new Button(By.XPath("//*[@class='select2-results__option select2-results__option--highlighted']"));
+
+    public TextInput LocationInput => new TextInput(By.XPath("//*[@class='select2-search__field']"));
+
+    public Label SkillsSection => new Label(By.XPath("//*[@class='default-label'][contains(text(), 'All Skills')]"));
+
+    public Dropdown SkillsDropdownPanel =>
+        new Dropdown(By.XPath("//*[@class='multi-select-dropdown' and @aria-hidden='false']"));
+
+    public Checkbox SkillOption(string skill) =>
+        new Checkbox(By.XPath($"//*[@class='checkbox-custom-label'][contains(text(), '{skill}')]"));
+
+    public Label ChosenSkillFilter => new Label(By.XPath("//*[@class='filter-tag']"));
+
+    public Panel FiltersPanel => new Panel(By.Id("jobSearchFilterForm"));
+
+    public JobListingsPage()
+    {
+        _resultPage = new JobSearchResultPage();
+    }
+
+    public override bool IsOpened()
+    {
+        return Browser.Driver.GetUrl().Equals(UiTestSettings.JobListingUrl);
+    }
+
+    public override void OpenUrl()
+    {
+        Browser.Driver.GotToWebPageUrl(UiTestSettings.JobListingUrl);
+        Waiter.WaitForCondition(() => KeywordInputField.IsElementDisplayedOnPage());
+    }
+
+    public JobSearchResultPage SearchJobsByKeyword(string searchWord)
+    {
+        KeywordInputField.SendKeys(searchWord);
+        FindButton.Click();
+        Waiter.WaitForCondition(_resultPage.SearchResultHeadingTitle.IsElementDisplayedOnPage);
+        return new JobSearchResultPage();
+    }
+
+    public JobSearchResultPage SearchJobLocationsByKeyWord(string searchWord)
+    {
+        LocationsDropdown.Click();
+        Waiter.WaitForCondition(DropdownLocationsPanel.IsElementDisplayedOnPage);
+        LocationInput.SendKeys(searchWord);
+        ChosenCityLine.Click();
+        Waiter.WaitForCondition(SelectedLocationChosen(searchWord).IsElementDisplayedOnPage);
+        return new JobSearchResultPage();
+    }
+
+    public JobSearchResultPage SearchJobSkillsByKeyWord(string searchWord)
+    {
+        SkillsSection.Click();
+        Thread.Sleep(1300); // can't find locator for skillsDropdown panel that it could work without Thread.Sleep 
```

Comment:
You can remove "and @aria-hidden='false'" path from SkillsDropdownPanel locator try to use Waiter.WaitForCondition(() => !SkillsDropdownPanel.GetTextFromAttribute("class").Contains("hiden"));

Reply:
it helped! thanks!